Cargo.lock

```diff
@@ -284,7 +284,7 @@ dependencies = [
  "ipld_blockstore",
  "ipld_hamt",
  "lazy_static",
- "libp2p",
+ "libp2p 0.21.1",
```

Comment:
What was the reason for bumping versions? was there something that broke? (wondering if there is something to fix) I tried without the updates and things seemed fine

Reply:
Not so sure, I just did a cargo update and found this

Reply:
ah yeah, that's what I meant to ask, just wasn't sure why the cargo update was necessary